Dockerfile

```diff
@@ -62,6 +61,13 @@ RUN cd /usr/local/lvm2 \
 	&& make install_device-mapper
 # see https://git.fedorahosted.org/cgit/lvm2.git/tree/INSTALL
 
+# Install lxc
+RUN curl -sSL https://linuxcontainers.org/downloads/lxc/lxc-1.0.7.tar.gz | tar -v -C /usr/src/ -xz
```

Comment:
/me reads tar man pages,
brain explodes where the eff has this been all my life